modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/createLayout.js

```diff
@@ -0,0 +1,130 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+let resource = `<?xml version="1.0" encoding="UTF-8"?>
+                <layout xmlns="urn:enonic:xp:model:1.0">
+                  <display-name i18n="key.display-name">Virtual Layout</display-name>
+                  <description i18n="key.description">My Layout Description</description>
+                  <form>
+                    <input type="Double" name="pause">
+                      <label i18n="key1.label">Pause parameter</label>
+                      <immutable>false</immutable>
+                      <indexed>false</indexed>
+                      <help-text i18n="key1.help-text"/>
+                      <occurrences minimum="0" maximum="1"/>
+                    </input>
+                    <item-set name="myFormItemSet">
+                      <label>My form item set</label>
+                      <immutable>false</immutable>
+                      <occurrences minimum="0" maximum="1"/>
+                      <items>
+                        <input type="TextLine" name="myTextLine">
+                          <label>My text line</label>
+                          <immutable>false</immutable>
+                          <indexed>false</indexed>
+                          <occurrences minimum="1" maximum="1"/>
+                        </input>
+                        <input type="TextLine" name="myCustomInput">
+                          <label>My custom input</label>
+                          <immutable>false</immutable>
+                          <indexed>false</indexed>
+                          <occurrences minimum="0" maximum="1"/>
+                        </input>
+                        <field-set>
+                          <label>My field set</label>
+                          <items>
+                            <input type="TextLine" name="fieldSetItem">
+                              <label i18n="key2.label">Field set Item</label>
+                              <immutable>false</immutable>
+                              <indexed>false</indexed>
+                              <help-text i18n="key2.help-text"/>
+                              <occurrences minimum="0" maximum="1"/>
+                            </input>
+                          </items>
+                        </field-set>
+                      </items>
+                    </item-set>
+                  </form>
+                  <regions>
+                    <region name="header"/>
+                    <region name="main"/>
+                    <region name="footer"/>
+                  </regions>
+                </layout>
+                `
+
+// BEGIN
+// Create virtual layout.
+var result = schemaLib.createComponent({
+    key: 'myapp:mylayout',
+    type: 'LAYOUT',
+    resource: resource
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
    resource
```